src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -81,6 +81,15 @@ public PersonCard(Person person, int displayedIndex) {
             vBox.getChildren().add(address);
         }
 
+        if (person.hasNonEmptyDescription()) {
+            Label description = new Label(person.getDescription()
+                .map(Object::toString)
+                .orElse(null));
```

Comment:
perhaps can use `.orElseThrow()` to throw early.

Reply:
Thanks for the suggestions! I think this change is trivial so I will just leave this as it is, because this part is copied from implementation of other fields so I think is fine to leave it for consistency purpose